api/docgen/openrpc.go

```diff
@@ -92,7 +92,7 @@ func ParseCommentsFromNodebuilderModules(moduleNames ...string) (Comments, Comme
 	return nodeComments, permComments
 }
 
-func NewOpenRPCDocument(comments Comments, permissions Comments) *go_openrpc_reflect.Document {
+func NewOpenRPCDocument(comments, permissions Comments) *go_openrpc_reflect.Document {
```

Comment:
Why though? I like that you dont need to specify the  same type over in parameters. Can we disable this rule?